rust/template/src/lib.rs

```diff
@@ -61,6 +61,10 @@ use serde::Deserializer;
 use serde::Serialize;
 use serde::Serializer;
 
+// This import is only needed to convince the OS X compiler to export
+// `extern C` functions declared in ddlog_log.rs in the generated lib.
+pub use ddlog_log as hidden_ddlog_log;
```

Comment:
```suggestion
#[doc(hidden)]
pub use ddlog_log;
```
A much better way of hiding extraneous documentation, it'll clean up the rustdoc output and will help to not confuse users

Reply:
Thanks, I'll add `doc(hidden)`, but renaming is still necessary to avoid conflict with another import below.